plugins/scaffolder/src/components/fields/RepoUrlPicker/RepoUrlPicker.tsx

```diff
@@ -157,9 +158,17 @@ export const RepoUrlPicker = (props: RepoUrlPickerProps) => {
 
   const hostType =
     (state.host && integrationApi.byHost(state.host)?.type) ?? null;
-
   return (
     <>
+      {schema.title && (
+        <Box my="8px">
```

Comment:
does it look bad without the hardcoded margin? Have you tried using some responsive margin with a `number` value?

Reply:
you are right, what about this?

Reply:
nice!